counter/src/main/java/edu/nextstep/camp/counter/CounterViewModel.kt

```diff
@@ -0,0 +1,34 @@
+package edu.nextstep.camp.counter
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class CounterViewModel: ViewModel() {
+    private val _count = MutableLiveData<Long>()
+    val count: LiveData<Long>
+        get() = _count
```

Comment:
아래 처럼 작성할 수도 있습니다 :)
```suggestion
    private val _count = MutableLiveData<Long>()
    val count: LiveData<Long> = _count
```